web/components/contract/boost-button.tsx

```diff
@@ -351,25 +351,34 @@ function FeedAnalytics(props: { contractId: string }) {
         .eq('contract_id', contractId)
   )
 
+  if (adQuery.error || viewQuery.error) {
```

Comment:
You can't early return if there's a react hook being used below: https://stackoverflow.com/a/56976469

I think just consolidate the error case like before and we should be good.